package.json

```diff
@@ -29,9 +29,12 @@
     "@material-ui/core": "^4.9.1",
     "@material-ui/icons": "^4.9.1",
     "@material-ui/lab": "4.0.0-alpha.45",
+    "@types/node-fetch": "^2.5.7",
+    "cross-fetch": "^3.0.6",
     "date-fns": "^2.15.0",
     "history": "^5.0.0",
     "moment": "^2.27.0",
+    "node-fetch": "^2.6.1",
```

Comment:
This and the @types aren't used, I think.

Reply:
Done